src/txdb.cpp

```diff
@@ -104,31 +104,19 @@ bool CCoinsViewDB::GetStats(CCoinsStats &stats) const {
     /* It seems that there are no "const iterators" for LevelDB.  Since we
        only need read operations on it, use a const-cast to get around
        that restriction.  */
-    boost::scoped_ptr<leveldb::Iterator> pcursor(const_cast<CLevelDBWrapper*>(&db)->NewIterator());
-    pcursor->SeekToFirst();
+    std::unique_ptr<CLevelDBIterator> pcursor(const_cast<CLevelDBWrapper*>(&db)->NewIterator());
```

Comment:
Was the change to unique_ptr to remove a warning?

Reply:
We got unique_ptr with C++11, I prefer to use std when possible. Any reason to use scoped_ptr over unique_ptr?